srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
srun: Warning: can't run 1 processes on 4 nodes, setting nnodes to 1
2160
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
2160
2160
2160
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: Terminating job step 9431642.12
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: Terminating job step 9431642.14
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: Terminating job step 9431642.9
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: Terminating job step 9431642.13
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: Terminating job step 9431642.10
srun: error: sh02-01n06: task 0: Exited with exit code 1
srun: Terminating job step 9431642.11
srun: error: sh02-01n02: task 0: Exited with exit code 1
srun: Terminating job step 9431642.1
srun: error: sh02-01n02: task 0: Exited with exit code 1
srun: Terminating job step 9431642.2
srun: error: sh02-01n02: task 0: Exited with exit code 1
srun: Terminating job step 9431642.0
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
    main()
  File "run_linearization_heuristic.py", line 99, in main
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
    delta = all_instances[instance_index][0]
IndexError: list index out of range
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
Traceback (most recent call last):
    delta = all_instances[instance_index][0]
  File "run_linearization_heuristic.py", line 871, in <module>
IndexError: list index out of range
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: Terminating job step 9431642.28
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: Terminating job step 9431642.24
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: Terminating job step 9431642.26
srun: Terminating job step 9431642.16
srun: Terminating job step 9431642.25
srun: Terminating job step 9431642.15
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: Terminating job step 9431642.27
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: Terminating job step 9431642.21
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: Terminating job step 9431642.29
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: Terminating job step 9431642.19
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: Terminating job step 9431642.20
srun: Terminating job step 9431642.23
srun: Terminating job step 9431642.17
srun: Terminating job step 9431642.18
srun: error: sh02-01n07: task 0: Exited with exit code 1
srun: Terminating job step 9431642.22
2160
2160
2160
2160
2160
2160
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
Traceback (most recent call last):
  File "run_linearization_heuristic.py", line 871, in <module>
    main()
  File "run_linearization_heuristic.py", line 99, in main
    delta = all_instances[instance_index][0]
IndexError: list index out of range
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: Terminating job step 9431642.8
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: Terminating job step 9431642.7
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: Terminating job step 9431642.5
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: error: sh02-01n04: task 0: Exited with exit code 1
srun: Terminating job step 9431642.4
srun: Terminating job step 9431642.6
srun: Terminating job step 9431642.3
